shard.yml

```diff
@@ -0,0 +1,8 @@
+name:
+version: 0.1.0
+author: Luis Ferreira
+
+dependencies:
+  kemal:
+    github: sdogruyol/kemal
+    branch: master
```

Comment:
Just looked into [shards](https://github.com/ysbaddaden/shards) docs and seems like we can specify a version here:

```yaml
dependencies:
  kemal:
    githun: sdogruyol/kemal
    version: ~> 0.5.0
```

Thanks @sdogruyol . Will update.